scripts/qa_crowbarsetup.sh

```diff
@@ -1267,6 +1267,14 @@ EOF
     # setup_base_images.rb is for SUSE Cloud 1.0 and update_nodes.rb is for 2.0
     sed -i -e 's/\(rootpw_hash.*\)""/\1"$2y$10$u5mQA7\/8YjHdutDPEMPtBeh\/w8Bq0wEGbxleUT4dO48dxgwyPD8D."/' /opt/dell/chef/cookbooks/provisioner/recipes/setup_base_images.rb /opt/dell/chef/cookbooks/provisioner/recipes/update_nodes.rb
 
+    if [[ $hacloud = 1 ]] ; then
+        f=/opt/dell/chef/cookbooks/nfs-server/templates/default/exports.erb
+        mkdir -p /var/lib/glance/images
+        if ! grep -q /var/lib/glance/images $f; then
+            echo "/var/lib/glance/images     <%= @admin_subnet %>/<%= @admin_netmask %>(rw,async,no_root_squash,no_subtree_check)" >> $f
```

Comment:
A customer would use a proper NFS server (or another way of shared storage), not a nfs share on the admin server (and it's definitely not something we'd add to the product).

But we don't have that proper shared storage in mkcloud, so we use some on the admin server NFS share.  Would you have a better recommendation?